### ISSUE TITLE
[tvstore] Exception (tvstore): Index was outside the bounds of the array.

### ISSUE THREAD

**Issue by garfield69** [Jackett/Jackett#7876]
### Environment
**OS**: windows 10 home
**.Net Runtime**: .Net-Core
**Jackett Version**: 0.14.299
**Last Working Jackett Version**: 0.14.287
**Are you using a proxy or VPN?** no
### Description
testing tvstore results in the error 
 **Exception (tvstore): Index was outside the bounds of the array.**
It is likely due to #7342 commit b5a6aab894d45c2d43f71e4db3452e5932d77fa7

I did try out on visual studio 2019 just seeing if replacing -1 with 1 fixed it, but the problem turned into a Exception (tvstore): Index was outside the bounds of the array.: Index was outside the bounds of the array, 
at which point I gave up.
### Logged Error Messages
[log.txt](https://github.com/Jackett/Jackett/files/4392668/log.txt)

[edit] I can provide temp access if required.

**Comment by cadatoiva** [comment]
So based on the commit change, it would appear that the error you get when changing -1 back to 1 was suppressed (on purpose) by the try block. This isn't a very good practice, but it's an easy fix for now to put back they try/catch. That tracker is in bad need of updating to standard methods, so at some point when I have some time temp access will be good.

**Comment by garfield69** [comment]
@cadatoiva  email sent.

**Comment by cadatoiva** [comment]
Thanks, this is lower on the priority change since the PR above will resolve the issue for now. I'm working through any other immediate concerns that don't have a fix now. It'll be a bit slower since I'm on my laptop instead of my main developing rig, but I'm working on them.

**Comment by ngosang** [comment]
@cadatoiva please, give some priority to this one. I hate seeing warnings.

**Comment by cadatoiva** [comment]
That's weird, I thought I PR'd this for the basic fix which is what I was referring to in the previous comment. Let me push that quick.

**Comment by cadatoiva** [comment]
Wait, now I'm confused actually. Yes, I can restore the `-1` -> `1`, and the try-catch block, but the try-catch block doesn't suppress anything, so the error you're seeing after fixing the -1 would have been there before this update. @garfield69 can you double check on functionality prior to 14.297?

**Comment by garfield69** [comment]
so to be clear, you want me to download build 0.14.287 and check if the tvstore indexer works or generates an error?

**Comment by garfield69** [comment]
The tvstore indexer from build 0.14.287 passes the test...

**Comment by garfield69** [comment]
F**k. I just repeated my source code amendment using visual studio 2019, by simply changing the -1 to 1 on the current github source, but this time, just to be sure, I also did a Clear-Build and then started a debug run.
And this time the tvstore test was successful.
So I apologise, looks like I screwed up the test somehow.

so using the existing 0.13.304 source and simply changing the -1 to 1 does work.
I will submit a push and I will close this ticket.
